docs/howto-view-telemetry-in-applicationinsights.md

```diff
@@ -0,0 +1,56 @@
+# How to view logs and metrics in Application Insights
```

Comment:
Should go into tutorials folder as tut-appinsights.md

Link to this from the readme.md in tutorials (second level, where the bullet points are)